### DISCUSSION TITLE
800A03EA 语法错误

### DISCUSSION THREAD

**Discussion by DogeCA** [1715173329/UnblockNeteaseMusic#604] [Q&A]
用npm安装后使用npx运行，出现如上错误提示，用release的exe则没有问题
![微信截图_20220330120006](https://user-images.githubusercontent.com/93893655/160748796-f5fcd779-563e-4d69-8dd3-c3a9e366e584.png)

**Comment by pan93412** [comment]
不要用 WScript 開啟；請改用 Node.js Runtime 啟動。不是很確定為何你啟動的是 wscript。
